content/en/docs/developer/mappers/_index.md

```diff
@@ -10,7 +10,24 @@ toc: true
 type: docs
 ---
 
+## Overview
+Mapper is an application that is used to connect and control devices. Following are the responsibilities of mapper:
```

Comment:
“Following are the responsibilities of mapper” seems a little weird. Maybe "The responsibilities of mapper are as follows" looks better.

Reply:
Updated it!